### ISSUE TITLE
Add support for data imports from TSDB 

### ISSUE THREAD

**Issue by bharathappali** [kruize/autotune#1006]
Kruize currently has implementation for posting data to it via update results API, adding data import from Timeseries databases (TSDB's) like prometheus, thanos, influx db, victoria metrics etc can help kruize to be used in live environments along with remote environments.

Steps that might involve to add this feature:

- [ ] [Finalize and document the request and responses for new API's in API doc](https://github.com/kruize/autotune/issues/1007)
- [ ] Add specific API's in kruize for accepting the TSDB config
- [ ] Proof of concept working functionality of importing data and auto-create experiments
- [ ] Integrate EM's trial manager mechanism to timely import data 
- [ ] Integrate Kruize recommendation engine to generate recommendation from the data imported 

Once this feature is ready in a POC mode, Kruize can implement it in the production version by creating a production level code with necessary abstractions

**Comment by bharathappali** [comment]
Closing this issue as this POC is not a priority. The main goal of this issue is treated a new feature of bulk api.
